### ISSUE TITLE
Proposing `Doxygen Docs` for NS3-GUI-HELPER

### ISSUE THREAD

**Issue by 20CE034** [NetworkingDevs/NS3-GUI-HELPER#15]
Example Docs of How-much-of-NEWS-are-you
it requires basic installation steps, refer screenshot below
![image](https://github.com/NetworkingDevs/NS3-GUI-HELPER/assets/94553525/062264f6-6a02-408d-a702-285dc65c5dcd)

https://www.doxygen.nl/

**Comment by HenilMistry** [comment]
Thank you for showing off with this tool! Hopefully it will add great benefit to this project!

**Comment by HenilMistry** [comment]
Yeah, I totally agree with you. Other than doxygen, what are your thoughts on using

1. Javadoc Tool
2. NaturalDocs
3. Sphinx
4. DocFx
5. ROBODoc
6. Or some markdown based tools

**Comment by 20CE034** [comment]
ns3's doxygen documentation adoption is good since they are also using it - https://www.nsnam.org/doxygen/ making this software follow a standardized docs.

**Comment by HenilMistry** [comment]
Sure then, we'll match with the NS3.
